src/main/java/org/jenkinsci/plugins/gitserver/FileBackedHttpGitRepository.java

```diff
@@ -39,19 +38,28 @@ public abstract class FileBackedHttpGitRepository extends HttpGitRepository {
      * Directory of the local workspace on the controller.
      * There will be "./.git" that hosts the actual repository.
      */
-    public final File workspace;
+    public final Path workspace;
```

Comment:
Caused [JENKINS-75401](https://issues.jenkins.io/browse/JENKINS-75401).

Reply:
Thanks for fixing it, @basil.